src/App.tsx

```diff
@@ -21,9 +37,10 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={handlePerPage}
           >
             <option value="3">3</option>
-            <option value="5">5</option>
+            <option value="5" selected>5</option>
```

Comment:
Instead of using the 'selected' attribute, set the initial value of the select element using the 'value' attribute of the select element. This will prevent the warning in the console. 🌟